spec/presenters/service_standard_presenter_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe ServiceStandardPresenter, "#content_id" do
+  it "is a preassigned UUID" do
```

Comment:
That one doesn't bother me personally but I've updated it to use 'returns'. :+1: